openslides_backend/action/actions/motion/set_support_self.py

```diff
@@ -28,6 +33,16 @@ class MotionSetSupportSelfAction(MeetingUserHelperMixin, UpdateAction):
     permission_id = "motion_id"
     permission = Permissions.Motion.CAN_SUPPORT
 
+    def check_permissions(self, instance: dict[str, Any]) -> None:
+        if not len(
```

Comment:
The `if` clause is redundant, right? The helper method will throw an exception if the delegator restriction applies.

Reply:
It's necessary to retain old functionality